src/wd.rs

```diff
@@ -1,15 +1,19 @@
 //! WebDriver types and declarations.
 
 use crate::error;
+use serde::{Deserialize, Serialize};
 use std::borrow::Cow;
 use std::convert::TryFrom;
 use std::fmt;
+use std::time::Duration;
+use webdriver::command::TimeoutsParameters;
 
 /// A [handle][1] to a browser window.
 ///
 /// Should be obtained it via [`Client::window()`] method (or similar).
 ///
 /// [1]: https://www.w3.org/TR/webdriver/#dfn-window-handles
+/// [`Client::window()`]: crate::Client::window
```

Comment:
You can avoid the need for all of these, I think, by adding this to the `use` list:
```rust
#[cfg(doc)]
use crate::Client;
```